server_http/static/main_script.js

```diff
@@ -557,9 +620,15 @@ message_element.onpaste = function (e) {
     }
 }
 
+
+share_button.onclick = copy_chat_url;
+
 all_messages_element.addEventListener('scroll', activate_scroll_event);
 document.addEventListener('click', read_message);
 document.addEventListener('keypress', read_message);
 
 document.onclick = deactivate_on_focus;
 document.onkeydown = deactivate_on_focus;
+
+// todo fix special characters (e.g., "_") in chat names which are displayed but break active users list
```

Comment:
create a github issue to not forget about it

Reply:
it's already there
https://github.com/KajaBraz/ChatBox/issues/52